lineage/states/StateDistributionGamma.py

```diff
@@ -190,3 +189,46 @@ def time_censor(cell, desired_experiment_time):
             # the daughters are no longer observed
             cell.left.observed = False
             cell.right.observed = False
+
+def atonce_estimator(x_list, gammas_list):
```

Comment:
You know what the input and output to this should look like. You should write this on a separate branch, and then write a test that checks it works.

Reply:
I moved this to another branch.